### PR TITLE
LPS-120932 Update alloyeditor to v2.11.14

### PR DESCRIPTION
**Pull request by wincent**
A release was requested in order to make [this fix](https://github.com/liferay/alloy-editor/pull/1425) available (related JIRA ticket: [LPS-120361](https://issues.liferay.com/browse/LPS-120361)).

Full [release notes](https://github.com/liferay/alloy-editor/releases/tag/v2.11.14).

Prevously tested here: https://github.com/liferay-frontend/liferay-portal/pull/372

### CODE REVIEW

**Comment by wincent** [comment]
ci:test:sf

**Comment by wincent** [comment]
ci:test:relevant

File: modules/yarn.lock [hunk 1]
```diff
@@ -4348,17 +4348,6 @@ babel-plugin-minify-type-constructors@^0.4.3:
   dependencies:
     babel-helper-is-void-0 "^0.4.3"
 
-babel-plugin-module-resolver@^3.2.0:
```
Comment (wincent) [inline modules/yarn.lock:4351]: I guess something got merged without properly updating the lockfile. So including this cleanup (didn't even need an `npx yarn-deduplicate yarn.lock`; just the `yarn add alloyeditor@2.11.14` was enough to clean out this cruft).

**Comment by wincent** [comment]
cc @NemethNorbert @julien

**Review by julien — APPROVED** [review]

**Review by julien — APPROVED** [review]

**Comment by julien** [comment]
ci:forward

**Pull request closed without merging**